[2024-09-22T07:41:02.784+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T07:41:02.793+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2024-09-21T00:00:00+00:00 [queued]>
[2024-09-22T07:41:02.797+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.is_api_available scheduled__2024-09-21T00:00:00+00:00 [queued]>
[2024-09-22T07:41:02.797+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-22T07:41:02.804+0000] {taskinstance.py:2888} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-09-21 00:00:00+00:00
[2024-09-22T07:41:02.808+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1089) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T07:41:02.809+0000] {standard_task_runner.py:72} INFO - Started process 1097 to run task
[2024-09-22T07:41:02.809+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'scheduled__2024-09-21T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp9l8zn35z']
[2024-09-22T07:41:02.810+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask is_api_available
[2024-09-22T07:41:02.835+0000] {task_command.py:467} INFO - Running <TaskInstance: user_processing.is_api_available scheduled__2024-09-21T00:00:00+00:00 [running]> on host 0bbd6d2129ab
[2024-09-22T07:41:02.872+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='is_api_available' AIRFLOW_CTX_EXECUTION_DATE='2024-09-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-21T00:00:00+00:00'
[2024-09-22T07:41:02.872+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-22T07:41:02.873+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-22T07:41:02.873+0000] {logging_mixin.py:190} INFO - Current task name:is_api_available state:running start_date:2024-09-22 07:41:02.793942+00:00
[2024-09-22T07:41:02.873+0000] {logging_mixin.py:190} INFO - Dag name:user_processing and current dag run status:running
[2024-09-22T07:41:02.873+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T07:41:02.873+0000] {baseoperator.py:405} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-09-22T07:41:02.873+0000] {http.py:137} INFO - Poking: api/
[2024-09-22T07:41:02.878+0000] {base.py:84} INFO - Retrieving connection 'user_api'
[2024-09-22T07:41:04.294+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2024-09-22T07:41:04.302+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-22T07:41:04.303+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, run_id=scheduled__2024-09-21T00:00:00+00:00, execution_date=20240921T000000, start_date=20240922T074102, end_date=20240922T074104
[2024-09-22T07:41:04.310+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-22T07:41:04.311+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-22T07:41:04.311+0000] {logging_mixin.py:190} INFO - Dag name:user_processing queued_at:2024-09-22 07:40:59.436225+00:00
[2024-09-22T07:41:04.311+0000] {logging_mixin.py:190} INFO - Task hostname:0bbd6d2129ab operator:HttpSensor
[2024-09-22T07:41:04.323+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-22T07:41:04.336+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-22T07:41:04.337+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
